src/ripple/app/main/Application.cpp

```diff
@@ -844,7 +854,9 @@ class ApplicationImp
 
         mValidations->flush ();
 
-        m_overlay->saveValidatorKeyManifests (getWalletDB ());
+        validators_->stop ();
+
+        manifestCache_->save (getWalletDB ());
```

Comment:
We should not use `wallet.db`. The manifest cache should have its own file, like peerfinder (`peerfinder.sqlite`).